assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -81,6 +97,15 @@ export default Ember.Controller.extend({
     this.set("loading", true);
     if (this.get("selectedItem.description") === "")
       this.set("selectedItem.description", "");
+
+    // group_ids must not be an empty array, as the param will not be sent the server.
```

Comment:
I don't like using the same variable to represent two different actions. Rather than using a magic `-1` value to represent this I would far prefer if you added a second parameter that indicates "remove all".

Reply:
Great idea. I will do that.